src/hermes/commands/workflow.py

```diff
@@ -107,13 +109,25 @@ def process():
 
 
 @click.group(invoke_without_command=True)
-def deposit():
+@click.option("--auth-token", envvar="HERMES_DEPOSITION_AUTH_TOKEN")
+@click.pass_context
+def deposit(click_ctx: click.Context, auth_token):
     """
     Deposit processed (and curated) metadata
     """
     click.echo("Metadata deposition")
     _log = logging.getLogger("cli.deposit")
 
+    # TODO: Better name than session?
+    # TODO: If this is needed in more places, it could be moved one level up.
+    click_ctx.session = requests.Session()
+    click_ctx.session.headers = {
+        "User-Agent": hermes_user_agent,
+    }
+    # TODO: This seems weird. Is there an option to put it directly into the context
+    # instead of as an argument?
+    click_ctx.auth_token = auth_token
```

Comment:
**note:** It should be available as `click_ctx.params["auth_token"]` (or similar, I guess).

Reply:
Yes, thank you! fcd01ea9d58fcb168173a33009840a5e4212828e